src/tv2-common/parts/effekt.ts

```diff
@@ -172,44 +192,55 @@ export function CreateEffektForPartInner<
 	}
 }
 
-export function CreateMixForPartInner(
+export function CreateMixEffectBlueprintPieceForPart(
```

Comment:
it is called `Create[...]BlueprintPiece[...]`, but what it really does is returning the `inTransition`, whereas creating the piece is kind of a side effect. Also `MixEffect` is unfortunate and confusing because Mix Effect (M/E) is a part of the switcher. We should refer to them as `MixTransition`, `DipTransition`...

Reply:
Yes, that was the behaviour before my change and I decided to keep to avoid too much refactoring.
But, as you point out, sideeffects are wrong, so I'm refactoring it now :) 

Refactoring the names to be `CreateMixTransitionBlueprintPieceForPart` and same for the Dip method